### PR TITLE
Update icra-2-nongaussian.ipynb

### PR DESCRIPTION
**Pull request by dehann**
xref
- #13 
- #14

### CODE REVIEW

**Comment by dehann** [comment]
Oops, I may have broken the json syntax somewhere

File: julia/caesar-api/icra-2-nongaussian.ipynb [hunk 1]
```diff
@@ -38,6 +38,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "# import Pkg; Pkg.add(Pkg.PackageSpec(name=\"IncrementalInference\", version=\"0.27\"))"
```
Comment (dehann) [inline julia/caesar-api/icra-2-nongaussian.ipynb]:
```suggestion
    "# import Pkg; Pkg.add(Pkg.PackageSpec(name=\"IncrementalInference\", version=\"0.27\"))",
```

**Comment by GearsAD** [comment]
Looks good - think the IncrementalInference is included correctly and you don't need to install it. I tried quickly and seems good. But let's leave it in and if you're happy can remove the issue.

image

**Comment by dehann** [comment]
> But let's leave it in and if you're happy can remove the issue.

cool, I put the IIF install in as a comment in case as user might need it.  I'd say keep the comment line for now and close the issue :-)

**Pull request merged**